workloads/router-perf-v2/common.sh

```diff
@@ -27,6 +27,15 @@ export NUMBER_OF_ROUTERS=${NUMBER_OF_ROUTERS:-2}
 export CERBERUS_URL=${CERBERUS_URL}
 export SERVICE_TYPE=${SERVICE_TYPE:-NodePort}
 
+if [[ ${COMPARE_WITH_GOLD} == "true" ]]; then
```

Comment:
@mohit-sheth Quick question, where do we call the comparison script? I might be missing something.

Reply:
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/a10977e2d86ec49bea95fb41b78df50bbabfc473/workloads/router-perf-v2/ingress-performance.sh#L35